### PR TITLE
Fix Fedora based CI on 5.16 and on Archlinux with glibc 2.35

### PR DESCRIPTION
**Pull request by adrianreber**
5.16 has changes which break CRIU when it comes to BPF and file maps.

This handles a new BPF related field in fdinfo and removes a flock test which is no longer supported.

### CODE REVIEW

**Review by rst0git — APPROVED** [review]
LGTM

**Comment by adrianreber** [comment]
The failing CentOS 8 tests are tracked here: #1734

File: images/bpfmap-file.proto [hunk 1]
```diff
@@ -21,4 +21,5 @@ message bpfmap_file_entry {
 	required string		map_name	= 13;
 	required uint32		ifindex		= 14 [default = 0];
 	optional sint32		mnt_id		= 15 [default = -1];
+	optional uint64		map_extra	= 16;
```
Comment (avagin) [inline images/bpfmap-file.proto:24]: Is it zero by default? Should we return a warning if it isn't zero? We have to be sure that we restore mappings properly.

Comment (adrianreber) [reply images/bpfmap-file.proto:24]: Added a warning if it is non-zero.

**Review by mihalicyn — APPROVED** [review]
LGTM

**Pull request merged**